README.md

```diff
@@ -18,7 +18,40 @@ None.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
+| clamav_clamd_conf | A dictionary of values to set in the clamd configuration file. | `{}` | No |
```

Comment:
Since we have `clamav_configuration_backup` I would rather use the long form for consistency.

```suggestion
| clamav_clamd_configuration | A dictionary of values to set in the clamd configuration file. | `{}` | No |
```